src/openrct2/core/File.h

```diff
@@ -27,4 +28,5 @@ namespace File
     bool Move(const std::string &srcPath, const std::string &dstPath);
     void * ReadAllBytes(const std::string &path, size_t * length);
     void WriteAllBytes(const std::string &path, const void * buffer, size_t length);
+    std::vector<std::string> ReadAllLines(const std::string &path);
```

Comment:
Why is it `File.h` and not `File.hpp`? I recall you provided a reason at some point, but I don't remember it.